Makefile

```diff
@@ -1,29 +1,27 @@
 
-
 # define VERBOSE to see executed commands
-
+# default build configuration
 TARGET=hyped
-MAIN=main.cpp
+MAIN=run/main.cpp
```

Comment:
This way we can separate all the "content" cpp files from the main cpp file to run. Since the main file dictates the functionality of application by instantiating modules it needs, many different mains can be created, i.e. demos in the last year repo.

The separations has two advantages:
1. it is clear that main is different from other cpp files, and it also shows how other mains can be constructed (demos)
2. it allows us to compile all object files separately from the main entry point; this is necessary for testing as the testrunner has its own main entry point that would give `multiple definitions of main symbol` linker error if trying to link everything together